packages/vue/src/TolgeeProvider.ts

```diff
@@ -18,28 +20,69 @@ export const TolgeeProvider = defineComponent({
     fallback: {
       type: [Object, String] as PropType<JSX.Element | string>,
     },
+    staticData: {
+      type: Object as PropType<TolgeeStaticData>,
+      default: undefined,
+    },
+    language: {
+      type: String as PropType<string>,
+      default: 'en',
```

Comment:
Opinion: I wouldn't put a default value here. I'd rather throw an exception in case user doesn't provide the language and the ssr is enabled. So we hint him that he needs to resolve the language on his own.

This way it will be just set to `en`, but he won't know where to change it, since normally it's done through tolgee instance.

Reply:
Oops, this value was for testing only. I will remove it